ldi-core/ngsiv2-to-ld/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/valuobjects/LinkedDataModel.java

```diff
@@ -0,0 +1,130 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.valuobjects;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.valuobjects.properties.LinkedDataAttribute;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.valuobjects.valueproperties.DateTimeValue;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFParser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.config.NgsiV2ToLdMapping.*;
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.services.NgsiLdURIParser.toNgsiLdUri;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class LinkedDataModel extends LinkedDataAttributeBase {
+
+	private List<String> contexts;
+	private String id;
+	private String type;
+	private DateTimeValue dateCreated;
+	private DateTimeValue dateModified;
+	private DateTimeValue dateObserved;
+
+	public LinkedDataModel() {
+		super();
+		this.contexts = new ArrayList<>();
+	}
+
+	// deserialize from NGSI_V2
```

Comment:
Already good effort for documenting the API ! Best use /** for Javadoc. For simple // comments are not visible outside the class.